idunn/utils/default_settings.yaml

```diff
@@ -177,4 +177,13 @@ RECYCLING_MEASURES_MAX_AGE_IN_HOURS: 168 # 7 days (Older measures will be ignore
 
 ##########################
 ## Instant Answer
+IA_MAX_QUERY_LENGTH: 100
```

Comment:
Should we consider extending it to non IA queries? I'm especially thinking about direct queries (`?q=...`)

Reply:
Possibly. But direct queries to /maps/?q are forwarded to "/v1/autocomplete" by Erdapfel. So currently, there is nothing we can do in Idunn to handle them differently.